less/variables.less

```diff
@@ -269,6 +269,22 @@
 @spacing-3-tablet: 72px;
 @spacing-3-desktop: 96px;
 
+@spacing-4-mobile: 64px;
+@spacing-4-tablet: 96px;
+@spacing-4-desktop: 128px;
+
+@spacing-5-mobile: 80px;
+@spacing-5-tablet: 120px;
+@spacing-5-desktop: 160px;
+
+@spacing-6-mobile: 96px;
+@spacing-6-tablet: 144px;
+@spacing-6-desktop: 192px;
+
+@spacing-7-mobile: 112px;
+@spacing-7-tablet: 168px;
+@spacing-7-desktop: 224px;
```

Comment:
Do we need all of them?  Could we pick two?

Reply:
Yes, what about picking 5 and 7? Julian is agree with this